types/yargs/yargs-tests.ts

```diff
@@ -110,6 +110,23 @@ function line_count() {
         .argv;
 }
 
+// // ts4.1+ types only
+// function camelCase() {
```

Comment:
Can this test be uncommented?

Reply:
I had to bump the minimum camelcase version to 4.2, hopefully that's satisfactory.